### PR TITLE
drupal in a day workshop certificated generated

### PR DESCRIPTION
**Pull request by binson-b**

### CODE REVIEW

File: certificate/views.py [hunk 1]
```diff
@@ -2714,14 +2724,15 @@ def create_python_workshop_certificate(certificate_path, name, qrcode, type, pap
             else:
                 template = '3day_template_PWS2017Pcertificate'
 
+        # ws_date = '10 March 2018' or 'March 2018'
```
Comment (prathamesh920) [inline certificate/views.py]: Please remove commented code.

File: certificate/models.py [hunk 2]
```diff
@@ -172,6 +172,8 @@ class Drupal_WS(models.Model):
     name = models.CharField(max_length=200)
     email = models.EmailField()
     purpose = models.CharField(max_length=10, default='DRP')
+    status = models.CharField(max_length=10, default='Participated')
```
Comment (prathamesh920) [inline certificate/models.py]: ```status```  can be a Boolean instead of char.

File: certificate/models.py [hunk 3]
```diff
@@ -172,6 +172,8 @@ class Drupal_WS(models.Model):
...
+    ws_date = models.CharField(max_length=200, default='01 January 2016')
```
Comment (prathamesh920) [inline certificate/models.py]: Instead of  ```ws_date``` can we have simply ```date```
Also, it can be date field instead of char

**Review by prathamesh920 — CHANGES_REQUESTED** [review]

File: certificate/views.py (hunk 1 above)
Comment (binson-b) [reply certificate/views.py]: done

File: certificate/models.py (hunk 2 above)
Comment (binson-b) [reply certificate/models.py]: done

File: certificate/models.py (hunk 3 above)
Comment (binson-b) [reply certificate/models.py]: done

**Comment by ankitjavalkar** [comment]
Looks good

**Comment by prathamesh920** [comment]
Merging for now

**Pull request merged**